Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN mkdir -p /usr/src/protoc/ \
 # Install GRPC and Protobuf.
 RUN pip3 install --upgrade pip \
   && pip3 install \
-    grpcio==1.3.0 \
+    grpcio==1.3.5 \
```

Comment:
I am not sure whether the grpcio version here must match the one in artman requirements.txt file. If that is the case, let's add a comment.